utils.py

```diff
@@ -2125,47 +2135,37 @@ def num_denom(A, symbolic_expr=False):
         denom = [numpy.poly1d(denom) *
                  numpy.poly1d(A.matrix[0, j].denominator.coeffs)
                  for j in range(A.matrix.shape[1])]
-        num = [numpy.poly1d(num) *
-               numpy.poly1d(A.matrix[0, j].numerator.coeffs)
+        num = [numpy.poly1d(num)
+               * numpy.poly1d(A.matrix[0, j].numerator.coeffs)
                for j in range(A.matrix.shape[1])]
-        if symbolic_expr is True:
-            for n in range(len(denom)):
-                sym_den = (sym_den + denom[- n - 1] * s**n).simplify()
-            for n in range(len(num)):
-                sym_num = (sym_num + num[- n - 1] * s**n).simplify()
-            return sym_num, sym_den
-        else:
-            return num, denom
-
     elif type(A) == tf:
-        denom = []
-        num = []
 
         denom = [list(A.denominator.coeffs)[n] for n in range(
             len(list(A.denominator.coeffs)))]
         num = [list(A.numerator.coeffs)[n] for n in range(
             len(list(A.numerator.coeffs)))]
-        if symbolic_expr is True:
-            for n in range(len(denom)):
-                sym_den = (sym_den + denom[- n - 1] * s**n).simplify()
-            for n in range(len(num)):
-                sym_num = (sym_num + num[- n - 1] * s**n).simplify()
-            return sym_num, sym_den
-        else:
-            return num, denom
-"""
+
+    if symbolic_expr is True:
```

Comment:
We usually leave out equality to `True` and `False`. Just say `if symbolic_expr:`

Reply:
Yuck. I would have cried if I wrote this in C. Sorry, missed this in my review.